docs/docs/graphql-reference.md

```diff
@@ -4,112 +4,180 @@ title: GraphQL Query Options Reference
+          description
+        }
+      }
+    }
+  }
+}
+```
 
-Try using the editor's autocomplete (`Ctrl + Space`) to get extended details from the `packageJson` nodes.
+In the GraphiQL editor, try using the editor's autocomplete (`Ctrl + Space`) to get extended details from the `packageJson` nodes.
 
 If you're using Gatsby version `2.2.0` or later, you can remove `edges` and `node` from your query and replace it with `nodes`. The query will still work and the returned object will reflect the `nodes` structure.
 
-<iframe
-  title="A longer query with nodes"
-  src="https://graphql-reference-1124782374.gtsb.io/___graphql?query=%7B%0A%20%20allSitePlugin%20%7B%0A%20%20%20%20totalCount%0A%20%20%20%20nodes%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20version%0A%20%20%20%20%20%20%20%20packageJson%20%7B%0A%20%20%20%20%20%20%20%20%20%20description%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D&explorerIsOpen=false"
-  width="600"
-  height="400"
-></iframe>
+```graphql
+{
+  allSitePlugin {
+    totalCount
+    nodes {
+      name
+      version
+      packageJson {
+        description
+      }
+    }
+  }
+}
+```
 
 ## Limit
 
 There are several ways to reduce the number of results from a query. Here `totalCount` tells you there's 8 results, but `limit` is used to show only the first three.
 
-<iframe
-  title="Using limit"
-  src="https://graphql-reference-1124782374.gtsb.io/___graphql?query=%7B%0A%20%20allMarkdownRemark(limit%3A%203)%20%7B%0A%20%20%20%20totalCount%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20frontmatter%20%7B%0A%20%20%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&explorerIsOpen=false"
-  width="600"
-  height="400"
-  loading="lazy"
-></iframe>
+```graphql
+{
+  allMarkdownRemark(limit: 3) {
+    totalCount
+    edges {
+      node {
+        frontmatter {
+          title
+        }
+      }
+    }
+  }
+}
+```
 
 ## Skip
 
 Skip over a number of results. In this query `skip` is used to omit the first 3 results.
 
-<iframe
-  title="Using skip"
-  src="https://graphql-reference-1124782374.gtsb.io/___graphql?query=%7B%0A%20%20allMarkdownRemark(skip%3A%203)%20%7B%0A%20%20%20%20totalCount%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20frontmatter%20%7B%0A%20%20%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&explorerIsOpen=false"
-  width="600"
-  height="400"
-  loading="lazy"
-></iframe>
+```graphql
+{
+  allMarkdownRemark(skip: 3) {
+    totalCount
+    edges {
+      node {
+        frontmatter {
+          title
+        }
+      }
+    }
+  }
+}
+```
 
 ## Filter
 
 In this query `filter` and the `ne` (not equals) operator is used to show only results that have a title. You can find a good video tutorial on this [here](https://www.youtube.com/watch?v=Lg1bom99uGM).
```

Comment:
```suggestion
In this query the `filter` and `ne` (not equals) operators are used to show only results that have a title. You can find a good video tutorial on this [here](https://www.youtube.com/watch?v=Lg1bom99uGM).
```

Might be a non-issue (might be just me and 🇩🇪)?!

Reply:
I read this as: "the [modifier] operator" – i.e. "filter" is not an operator

Reply:
Ah, that makes sense of course!